packages/fxa-settings/src/lib/gql.test.ts

```diff
@@ -1,106 +1,60 @@
 /* This Source Code Form is subject to the terms of the Mozilla Public
  * License, v. 2.0. If a copy of the MPL was not distributed with this
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-import { errorHandler, pagesRequiringAuthentication } from './gql';
+import { errorHandler } from './gql';
 import { ErrorResponse } from '@apollo/client/link/error';
 import { Operation, NextLink, ServerError } from '@apollo/client/core';
 import { GraphQLError } from 'graphql';
-import Mock = jest.Mock;
+import { cache } from './cache';
+import { GET_LOCAL_SIGNED_IN_STATUS } from '../components/App/gql';
+import sentryMetrics from 'fxa-shared/lib/sentry';
 
 let errorResponse: ErrorResponse;
```

Comment:
nit: It seems odd to hoist the error reponse up here. Since we are cleaning this up, maybe we should fix this.